### PR TITLE
Add reversion history to API

### PR DESCRIPTION
**Pull request by szok**
Added generate reversion history to API action

### CODE REVIEW

**Comment by ghost** [comment]
:ok:

File: src/ralph/api/serializers.py [hunk 1]
```diff
@@ -188,6 +190,13 @@ def build_field(self, *args, **kwargs):
             field_kwargs.setdefault('style', {})['base_template'] = 'input.html'
         return field_class, field_kwargs
 
+    def save(self):
```
Comment (mkurek) [inline src/ralph/api/serializers.py]: please make sure that every class that is used in `save_serializer_class` is inheriting from `RalphAPISaveSerializer`

Comment (szok) [inline src/ralph/api/serializers.py]: Yes, because all viewset inherit from RalphAPIViewSetMixin (RalphAPIViewSet)

Comment (mkurek) [inline src/ralph/api/serializers.py]: No, ex. `BaseObjectLicenceViewSet` and `BaseObjectLicenceSerializer` used in `save_serializer_class` - i assume that there is more cases like that.

Comment (szok) [inline src/ralph/api/serializers.py]: Done.

**Comment by ar4s** [comment]
:ok:

**Pull request merged**